rfd/0216-tbot-resiliency.md

```diff
@@ -0,0 +1,214 @@
+---
+authors: Dan Upton (daniel.upton@goteleport.com)
+state: draft
+---
+
+# RFD 216 - Improving `tbot`'s Resiliency for Production Workloads
```

Comment:
Another potential constraint: ideally whatever we choose here can be backported to v18 and v17. It's unlikely that key customers will be able to move to v18 for a couple of months.